<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>310</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>310</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>B-day | ADD PERSON</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>resources/logo.ico</normaloff>resources/logo.ico</iconset>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This form is needed to add a new person to your database&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>331</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FIRST NAME:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LAST NAME:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstName">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>170</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lastName">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>170</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DATE OF BIRTH:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backToMenu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>135</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>BACK TO MENU</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="bDate">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimumDate">
     <date>
      <year>1900</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
    <property name="maximumTime">
     <time>
      <hour>22</hour>
      <minute>59</minute>
      <second>59</second>
     </time>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::DaySection</enum>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="topGroup">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>50</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>8</y>
      <width>91</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>B-day</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
